fontforge/parsettf.c

```diff
@@ -4716,30 +4716,43 @@ return;
 	    len = getlong(ttf);
 	    /* language = */ getlong(ttf);
 	}
+	uint16 *dtable = NULL;
+	size_t dlen = len;
+	if (dlen >= 1048576) {
```

Comment:
How's this determined? Also looks like a magic number on first glance, but it's 2^20

Reply:
This is just a compromise between values seen in the wild (up to 120-thousand) and reasonable memory use. The assumption is that the file is malicious or corrupt if it goes over.